testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -581,7 +594,8 @@ impl BitcoinRegtestController {
                     received_at: Instant::now(),
                     state_transition
                 }
-            } else if let Some(height_to_wait) = height_to_wait {
+            }
+            else if let Some(height_to_wait) = height_to_wait {
```

Comment:
This is fine to leave as is in this PR, but I think we should use `rustfmt` to prevent these kinds of things going forward (https://github.com/blockstack/stacks-blockchain/issues/1907)

Reply:
Now using `rustfmt`.